qemu/cfg/host-kernel/host-kernel-check.cfg

```diff
@@ -19,54 +19,54 @@ variants:
         pre_check_cmd = "grep 'Red Hat Enterprise Linux' /etc/redhat-release"
         host_kernel_ver_str = Host_RHEL
         variants:
-            - 5:
+            - m5:
                 host_kernel_ver_str += ".5"
                 variants:
-                    - 7:
+                    - u7:
                         host_kernel_ver_str += ".7"
                         requires_kernel = [">= 2.6.18-274", "<  2.6.18-308"]
-                    - 8:
+                    - u8:
                         host_kernel_ver_str += ".8"
                         requires_kernel = [">= 2.6.18-308", "<  2.6.18-348"]
-                    - 9:
+                    - u9:
                         host_kernel_ver_str += ".9"
                         requires_kernel = [">= 2.6.18-348", "<  2.6.18-349"]
-                    - 10:
+                    - u10:
                         host_kernel_ver_str += ".10"
                         requires_kernel = [">= 2.6.18-349", "<  2.6.18-372"]
-                    - 11:
+                    - u11:
                         host_kernel_ver_str += ".11"
                         requires_kernel = [">= 2.6.18-372", "<  2.6.19"]
-            - 6:
+            - m6:
                 host_kernel_ver_str += ".6"
                 variants:
-                    - 1:
+                    - u1:
                         host_kernel_ver_str += ".1"
                         requires_kernel = [">= 2.6.32-131", "<  2.6.32-220"]
-                    - 2:
+                    - u2:
                         host_kernel_ver_str += ".2"
                         requires_kernel = [">= 2.6.32-220", "<  2.6.32-279"]
-                    - 3:
+                    - u3:
                         host_kernel_ver_str += ".3"
                         requires_kernel = [">= 2.6.32-279", "<  2.6.32-280"]
-                    - 4:
+                    - u4:
                         host_kernel_ver_str += ".4"
                         requires_kernel = [">= 2.6.32-280", "<  2.6.32-359"]
-                    - 5:
+                    - u5:
                         host_kernel_ver_str += ".5"
                         requires_kernel = [">= 2.6.32-359", "<  2.6.32-432"]
-                    - 6:
+                    - u6:
                         host_kernel_ver_str += ".6"
                         requires_kernel = [">= 2.6.32-432", "<  2.6.32-505"]
-                    - 7:
+                    - u7:
                         host_kernel_ver_str += ".7"
                         requires_kernel = [">= 2.6.32-505", "<  2.6.32-574"]
-            - 7:
+            - m7:
                 host_kernel_ver_str += ".7"
                 variants:
-                    - 0:
+                    - u0:
                         host_kernel_ver_str += ".0"
                         requires_kernel = [">= 3.2.1-0.8", "< 3.10.0-124"]
-                    - 1:
+                    - u1:
                         host_kernel_ver_str += ".1"
```

Comment:
Hi @spcui 

Just curious, could you explain why not update host_kernel_ver_str?

Thanks.